libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -559,12 +602,95 @@ AP_GPS_SBF::process_message(void)
         check_new_itow(temp.TOW, sbf_msg.length);
         RxState = temp.RxState;
         if ((RxError & RX_ERROR_MASK) != (temp.RxError & RX_ERROR_MASK)) {
-            GCS_SEND_TEXT(MAV_SEVERITY_INFO, "GPS %u: SBF error changed (0x%08x/0x%08x)", (unsigned int)(state.instance + 1),
+            GCS_SEND_TEXT(MAV_SEVERITY_INFO, "GPS %u: SBF internal error changed (0x%08x/0x%08x)", (unsigned int)(state.instance + 1),
                             (unsigned int)(RxError & RX_ERROR_MASK), (unsigned int)(temp.RxError & RX_ERROR_MASK));
         }
         RxError = temp.RxError;
+        if ((ExtError & EXT_ERROR_MASK) != (temp.ExtError & EXT_ERROR_MASK)) {
+            GCS_SEND_TEXT(MAV_SEVERITY_INFO, "GPS %u: SBF external error changed (0x%08x/0x%08x)", (unsigned int)(state.instance + 1),
+                            (unsigned int)(ExtError & EXT_ERROR_MASK), (unsigned int)(temp.ExtError & EXT_ERROR_MASK));
+        }
+        ExtError = temp.ExtError;
+#if AP_MAVLINK_MSG_GNSS_INTEGRITY_ENABLED
+        state.system_errors = static_cast<uint32_t>(AP_GPS::Errors::GPS_SYSTEM_ERROR_NONE);
+
+        if (ExtError & DIFFCORRERROR) {
+            state.system_errors |= static_cast<uint32_t>(AP_GPS::Errors::INCOMING_CORRECTIONS);
+        }
+
+        for (const auto &map_entry : sbf_error_map) {
+            if (RxError & map_entry.sbf_error_bit) {
+                state.system_errors |= static_cast<uint32_t>(map_entry.ap_error_enum);
+            }
+        }
+#endif  // AP_MAVLINK_MSG_GNSS_INTEGRITY_ENABLED
+        break;
+    }
+
+    case RFStatus:
+    {
+        const msg4092 &temp = sbf_msg.data.msg4092u;
+        check_new_itow(temp.TOW, sbf_msg.length);
```

Comment:
Again, change unrelated to new functionality.  I think this is probably benign, but it might just be neater to not know about the message when we're not doing integrity stuff?